pkg/apis/apps/v1alpha1/base.go

```diff
@@ -32,7 +32,8 @@ type Base struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	Spec BaseSpec `json:"spec"`
+	Spec   BaseSpec   `json:"spec"`
+	Status BaseStatus `json:"status,omitempty"`
```

Comment:
No need to update status to `Base`.

Reply:
This version designed syn the Desc to base's Status, subsequent tries to directly synchronize desc to subs

Reply:
Prefer directly sync status from Description to Subscription. From Description labels, you could find bound Subscriptions. You could also use label selectors to list all matching Decriptions.